lib/fireblocks/config.go

```diff
@@ -8,8 +8,8 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// options houses parameters for altering the behavior of a SimpleTxManager.
-type options struct {
+// Config houses parameters for altering the behavior of a SimpleTxManager.
+type Config struct {
```

Comment:
This was falsely named `options` - renaming it to fix the mistake. An option in this context is something which changes the config - addressed by introducing an `Option` type below and making it public so that it can be used by other packages, e.g. e2e.